assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -72,6 +82,26 @@ function HostDetails({
     )
   );
 
+  const dispatch = useDispatch();
+
+  const refreshCatalog = () =>
+    dispatch(
+      updateCatalog({
+        provider,
+        target_type: TARGET_HOST,
+      })
+    );
+
+  useEffect(() => {
```

Comment:
This is what I saw elsewhere; is there a more efficient way to call the Catalog here?

Reply:
Don't overcomplicate this, the browser is already architected to be efficient on the underlying level :+1: